internal/testutils/feature.go

```diff
@@ -109,3 +118,15 @@ func kernelVersion(tb testing.TB) internal.Version {
 	}
 	return v
 }
+
+func ignoreKernelVersionCheck(tb testing.TB) bool {
+	if tNames := os.Getenv(ignoreKernelVersionEnvVar); tNames != "" {
```

Comment:
Nit: you could exit early here instead: `if tNames == "" { return false }`

Reply:
sure.